myapp/config/application.rb

```diff
@@ -24,5 +24,16 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    config.generators do |g|
```

Comment:
Its weird to have this configuration in application.rb. Think that it will be also be included in the production.rb for production servers and tests shouldn't be there. Also I don't think this code is not necessary at all